test/README.md

```diff
@@ -0,0 +1,37 @@
+CSI Baremetal operator e2e testing
+---------------------
+
+Last update: 19.04.2021
+
+! Kind-cluster has pre-loaded csi images
+
+1. Set CSI version
+
+```
+export csiVersion=...
```

Comment:
Environment variables are defined with capital letters usually. Please use the following format for all env vars:
`export CSI_VERSION=<csi version>`

Reply:
done